src/graphql/NullConnection.ts

```diff
@@ -0,0 +1,25 @@
+export type NullConnectionType = {
+  edges: [];
+  count: number;
+  pageInfo: {
+    hasNextPage: boolean;
+    hasPreviousPage: boolean;
+    endCursor: string;
+    startCursor: string;
+  };
+  startCursorOffset: number;
+  endCursorOffset: number;
+};
+
+export const NullConnection: NullConnectionType = {
```

Comment:
I think NullConnection is not necessary anymore, you can return `null`

Reply:
Nice, thanks o/

Reply:
depends, connection types cannot be null, so you need to return a null connection with empty edges